lib/matplotlib/axis.py

```diff
@@ -1171,14 +1171,14 @@ def get_majorticklabels(self):
         ticks = self.get_major_ticks()
         labels1 = [tick.label1 for tick in ticks if tick.label1.get_visible()]
         labels2 = [tick.label2 for tick in ticks if tick.label2.get_visible()]
-        return cbook.silent_list('Text major ticklabel', labels1 + labels2)
+        return list(labels1 + labels2)
```

Comment:
you don't even need the `list` call here; `labels1 + labels2` is already a list.
same below.

Reply:
yeah, my mistake. corrected!